tee-worker/local-setup/launch-parachain.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python3
+"""
+Launch handily a local dev setup consisting of one integritee-node and some workers.
+
+Example usage: `./local-setup/launch.py /local-setup/simple-config.py`
```

Comment:
Is the comment up-to-date?
I see no difference between `launch-parachain.py` and `launch-worker.py` :(

Reply:
We don't need this, I was just testing things 😅 , Removed ✅